builder/docker-rules.mk

```diff
@@ -172,8 +174,13 @@ image_on_store: image_dep
 
 .PHONY: image_on_local
 image_on_local: image_dep $(EXPORT_DIR)rootfs.tar
+ifeq ($(USE_INTERNAL_HTTPD), 1)
+	$(eval LOCAL_HTTPD_PORT := $(shell for candidate in $$(seq 8000 8100); do lsof -i tcp:$$candidate >/dev/null 2>&1; if [ $$? = 1 ]; then echo $$candidate; break; fi; done))
+	(echo "HTTP/1.1 200 OK\r\nContent-type: application/x-tar\r\nContent-Disposition: attachment; filename=\"$(TARGET_UNAME_ARCH)-$(NAME)-$(VERSION).tar\"\r\nContent-Length: $(shell stat -c %s $(EXPORT_DIR)rootfs.tar)\r\nConnection: close\r\n\r" && cat $(EXPORT_DIR)rootfs.tar) | nc -l -p $(LOCAL_HTTPD_PORT) > /dev/null &
```

Comment:
Dash's `echo` doesn't handle **-e** switch, it just do it...
More about it [here](http://stackoverflow.com/questions/33784892/bash-vs-dash-behavior-with-the-command-echo-ne-hello-n).